CHANGES.md

```diff
@@ -1,3 +1,10 @@
+1.7.2 / 2017-MM-DD
```

Comment:
I took this idea from PHPUnit's changelog: https://github.com/sebastianbergmann/phpunit/commit/69f832b87c731d5cacad7f91948778fe98335fdd